lib/cli.ts

```diff
@@ -122,11 +123,21 @@ async function run() {
 
   program
     .command("deploy")
-    .description("Deploy the project (not implemented)")
-    .argument("[string]", "app name")
-    .action((appName) => {
-      console.log("not yet supported");
+    .description("Deploy the project")
+    .argument("[string]", "Workspace app name to deploy")
+    .option("--deploy-account-id <deployAccountId>", "Account under which component code should be deployed")
```

Comment:
This all looks good, can you run `bw help` and update the README with this new command?